src/Init/Prelude.lean

```diff
@@ -1731,19 +1715,22 @@ def Nat.ble : @& Nat → @& Nat → Bool
   | succ n, succ m => ble n m
 
 /--
-An inductive definition of the less-equal relation on natural numbers,
-characterized as the least relation `≤` such that `n ≤ n` and `n ≤ m → n ≤ m + 1`.
+Non-strict inequality of natural numbers, usually accessed via the `≤` operator.
 -/
 protected inductive Nat.le (n : Nat) : Nat → Prop
-  /-- Less-equal is reflexive: `n ≤ n` -/
+  /-- Non-strict inequality is reflexive: `n ≤ n` -/
   | refl     : Nat.le n n
   /-- If `n ≤ m`, then `n ≤ m + 1`. -/
   | step {m} : Nat.le n m → Nat.le n (succ m)
 
 instance instLENat : LE Nat where
   le := Nat.le
 
-/-- The strict less than relation on natural numbers is defined as `n < m := n + 1 ≤ m`. -/
+/--
+Strict inequality of natural numbers, usually accessed via the `<` operator.
+
+By definition, `n < m = n + 1 ≤ m`.
```

Comment:
```suggestion
It is defined as `n < m = n + 1 ≤ m`.
```
nit: I want it to be very clear that this is the definition, and not a property which follows from the definition